docs/src/to-contribute/boterinas.md

```diff
@@ -0,0 +1,60 @@
+# Bot for Asterinas 
+
+## Introduction
+
+`@boterinas` is a general-purpose bot designed for a wide variety of tasks in Asterinas. It streamlines maintenance tasks to enhance workflow efficiency. 
+
+Commands are issued by writing comments that start with the text `@boterinas`. The available commands depend on which repository you are using. The main Asterinas repository contains a `triagebot.toml` file where you can see which features are enabled. 
+
+Commands for GitHub issues or pull requests should be issued by writing `@boterinas` followed by the command anywhere in the comment. Note that `@boterinas` will ignore commands in markdown code blocks, inline code spans, or blockquotes. You can enter multiple `@boterinas` commands in a single comment.
+
+Additionally, `@boterinas` allows for editing comments. If you don't change the text of a command, the edit will be ignored. However, if you modify an existing command or add new ones, those commands will be processed.
+
+Below, you'll find a comprehensive guide on how to use `@boterinas` effectively.
+
+## Commands and Usage
+
+### Workflow Management
+- **`@boterinas rerun`**  
+  Restarts the workflow of the current pull request if it has failed unexpectedly. Only the author of the pull request can use this command.
+  *Example:* `@boterinas rerun`
+
+### Issue and Pull Request Management
+- **`@boterinas claim`**  
+  Assigns the issue or pull request to yourself.  
+  *Example:* `@boterinas claim`
+  
+- **`@boterinas release-assignment`**  
+  Removes the current assignee from an issue or pull request. This command can only be executed by the current assignee or a team member.  
+  *Example:* `@boterinas release-assignment`
+  
+- **`@boterinas assign @user`**  
+  Assigns a specific user to the issue or pull request. Only team members have permission to assign other users.  
+  *Example:* `@boterinas assign @some_asterinas`
+
+### Label Management
+- **`@boterinas label <label>`**  
+  Adds a label to the issue or pull request.  
+  *Example:* `@boterinas label C-bug`
```

Comment:
Most examples are useless: they just repeat the command format. The examples for the `label` command is probably the only ones that provide some extra information. I suggest removing useless examples and only the useful ones.

Reply:
Get it. I will replace the example with informative ones, such as `@boterinas label C-enhancement C-rfc` to show multiple labeling in one command.